conanfile.py

```diff
@@ -49,30 +72,46 @@
 target_link_libraries(lanelet2_python_compiler_flags INTERFACE lanelet2_core lanelet2_io lanelet2_routing lanelet2_traffic_rules lanelet2_projection lanelet2_matching)
 """
 
+
 def read_version():
-    package = ET.parse('lanelet2_core/package.xml')
-    return package.find('version').text
+    package = ET.parse("lanelet2_core/package.xml")
+    return package.find("version").text
```

Comment:
I think it would make sense to bump the version (at least the patch level) to notify pypi users that there is something new now :)

Reply:
Bumping the version and releasing is a semi-automated process. We can do that in a follow-up PR.